packages/notebook/src/browser/variables/inspector-script.ts

```
@@ -0,0 +1,349 @@
+  };
+
+  public static getScript(lang: string): Promise<LanguageModel> {
+    return new Promise((resolve, reject) => {
+      if (lang in Languages.scripts) {
+        resolve(Languages.scripts[lang]);
+      } else {
+        reject('Language ' + lang + ' not supported yet!');
+      }
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)